### ISSUE TITLE
I want to translate it into an HTML file just like before

### ISSUE THREAD

**Issue by myliwenbo** [KosmosisDire/obsidian-webpage-export#405]
I want to turn it into an HTML file just like before, that is, all the files are in this HTML and I don't need an external lib

Chinese：我想导成一个HTML文件和以前一样，就是所有的文件都在这个HTML里面并不需要外置的lib

**Comment by myliwenbo** [comment]
Is that all we need to turn off
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/29202248/1edb53ad-f45e-4038-b07a-bd17f29ec931)

**Comment by KosmosisDire** [comment]
No, all you have to do is switch the `Export Mode` option to HTML Documents.

See:
https://docs.obsidianweb.net/getting-started/5.-export-modes.html

**Comment by jvsteiner** [comment]
Th `lib/` folder is still produced, even when `Export Mode` option is set to `HTML Documents`.  I found that the file that is produced is larger, and it works on its own - ie. the lib folder can be deleted.  I think it's a bug that it is produced at all, if this is the intent is to produce a single-file html page.

**Comment by KosmosisDire** [comment]
Yes the lib folder should not be produced, I've fixed this in development but it will be available along with the next release
